portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +27,30 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  public ArrayList<String> coelho;
+
+  public void init(){
+      coelho = new ArrayList<String>();
+      coelho.add("I can choose either to be a victim of the world or an adventurer in search of treasure.");
+      coelho.add("Life is too short, or too long, for me to allow myself the luxury of living it so badly.");
+      coelho.add("When we strive to become better than we are, everything around us becomes better, too.");
+      coelho.add("At every moment of our lives, we all have one foot in a fairy tale and the other in the abyss.");
+      coelho.add("And, when you can\'t go back, you have to worry only about the best way of moving forward.");
+  }
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Ness!</h1>");
+        /*fetch-walkthrough*/
+    //response.setContentType("text/html;");
+    //response.getWriter().println("<h1>Hello Ness!</h1>\n");
+        /*JSON-walkthrough*/
+    //response.setContentType("application/json;");
+    //response.getWriter().println(convertToJsonUsingGson(coelho));
```

Comment:
Does /data show the quotes even though this line is commented out? (if so, just let me know and I'll approve the PR)

Reply:
The quotes show if those lines are uncommented (along with the div in index.html) but I have them commented out for now since otherwise they're pasted in the middle of the home page. Is that alright?
![image](https://user-images.githubusercontent.com/35279510/84474818-82845980-ac59-11ea-9cc6-40356b931b20.png)

Reply:
![image](https://user-images.githubusercontent.com/35279510/84474891-9def6480-ac59-11ea-80e3-66a036e378bb.png)

Reply:
Ah yeah, makes sense. Thanks for the screenshots! Those are always super helpful for reviewing, especially if you haven't deployed lately